internal/cli/config/config.go

```diff
@@ -14,53 +14,117 @@ See the Mulan PSL v2 for more details.
 package config
 
 import (
-	"bytes"
-	"fmt"
+	"errors"
+	"io"
+	"net/http"
 
-	"github.com/spf13/viper"
+	"gopkg.in/yaml.v2"
 
-	"github.com/oceanbase/ob-operator/internal/cli/generated/bindata"
 	"github.com/oceanbase/ob-operator/internal/cli/utils"
 )
 
-// component config for test
-var confPath = "internal/assets/cli-templates/component_config.yaml"
+// Chart struct to parse the chart.yaml
+type Chart struct {
+	apiVersion  string `yaml:"apiVersion"`
+	AppVersion  string `yaml:"appVersion"`
+	description string `yaml:"description"`
+	name        string `yaml:"name"`
+	chartType   string `yaml:"type"`
+	version     string `yaml:"version"`
+}
 
-func readComponentConf(path string) map[string]string {
-	components := make(map[string]string)
-	fileobj, err := bindata.Asset(path)
-	// panic if file not exists
+const (
+	stableVersion = "stable"
+	devVersion    = "master"
+)
+
+// ComponentList is the list of components that can be installed
+var ComponentList = []string{
+	"cert-manager", "ob-operator", "ob-dashboard", "local-path-provisioner", "local-path-provisioner-dev", "ob-operator-dev",
+}
+
+// ComponentUpdateList is the list of components that can be updated
+var ComponentUpdateList = []string{
+	"cert-manager", "ob-operator", "ob-dashboard", "local-path-provisioner",
+}
+
+var versionURLs = map[string]string{
+	"ob-dashboard":           "https://raw.githubusercontent.com/oceanbase/ob-operator/refs/heads/stable/charts/oceanbase-dashboard/Chart.yaml",
+	"local-path-provisioner": "https://raw.githubusercontent.com/rancher/local-path-provisioner/refs/tags/v0.0.30/deploy/chart/local-path-provisioner/Chart.yaml",
```

Comment:
Maybe we should use `master` branch here.

Reply:
As far as I see, the chart in the master branch is always of the stable version. The stable and dev are used to describe the status of deploy/local-path-storage.yaml, I thought.